deliver/lib/deliver/queue_worker.rb

```diff
@@ -6,9 +6,11 @@ module Deliver
   # Use this when you have all the items that you'll process in advance.
   # Simply enqueue them to this and call `QueueWorker#start`.
   class QueueWorker
+    NUMBER_OF_THREADS = Helper.test? ? 1 : [ENV.fetch("DELIVER_NUMBER_OF_THREADS", 10).to_i, 10].min
```

Comment:
I've extracted this from UploadScreenshots to reuse as the default value of its initialiser 🙂